.github/workflows/promote_charm.yaml

```diff
@@ -13,11 +13,17 @@ on:
         description: 'Destination Channel'
         options:
         - latest/stable
+    secrets:
+      CHARMHUB_TOKEN:
+        required: true
 
 jobs:
   promote-charm:
-    uses: canonical/operator-workflows/.github/workflows/promote_charm.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/promote_charm.yaml@promote-charm-base
```

Comment:
Reminder:
change canonical/operator-workflows/.github/workflows/promote_charm.yaml@promote-charm-base back to canonical/operator-workflows/.github/workflows/promote_charm.yaml@main once the PR on operator-workflows is merged.